custom_components/daikin_onecta/update.py

```
@@ -0,0 +1,167 @@
+from .const import ENTITY_CATEGORY
+from .const import TRANSLATION_KEY
+from .const import VALUE_SENSOR_MAPPING
+from .coordinator import OnectaDataUpdateCoordinator
+from .coordinator import OnectaRuntimeData
+from .device import DaikinOnectaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+# The Daikin Onecta cloud API exposes firmware info on the "gateway" management
+# point.  The relevant characteristics are:
+#   firmwareVersion       – currently installed version string
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)